packages/zmarkdown/common.js

```diff
@@ -206,26 +206,27 @@ function getDepth (node) {
 }
 
 module.exports = (
-  opts = {remarkConfig, rebberConfig},
-  target = 'html'
+  opts = {remarkConfig},
+  processor = null
 ) => {
-  if (!opts.remarkConfig || !Object.keys(remarkConfig).length) {
-    throw new Error(dedent`
-      This module expects to be called with ({remarkConfig, rebberConfig}, target = 'html'),
-      remarkConfig is missing!`)
+  if (!opts) {
+    opts = {}
   }
-  if (!opts.rebberConfig || !Object.keys(rebberConfig).length) {
-    throw new Error(dedent`
-      This module expects to be called with ({remarkConfig, rebberConfig}, target = 'html'),
-      rebberConfig is missing!`)
+
+  if (!opts.remarkConfig || (opts.remarkConfig === null) || !Object.keys(remarkConfig).length) {
```

Comment:
If `opts.remarkConfig === null`, then `!opts.remarkConfig === true`. ;)

Reply:
What I meant was: if `x` is `null`, then `!x` is always `true`.

A condition like `if (x === false || !x)` is redundant.

Reply:
It is changed.